manifests/a/Amazon/WorkspacesClient/5.9.0/Amazon.WorkspacesClient.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.2.5 $debug=QUSU.CRLF.7-3-3.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Amazon.WorkspacesClient
+PackageVersion: 5.9.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qb
+  Custom: /norestart
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://d2td7dqidlhjx7.cloudfront.net/prod/global/windows/Amazon+WorkSpaces.msi
+  InstallerSha256: 7D146F83B9398A38F04F606295916E3DD22D3D030749A7E5C1802846010C3A16
+  ProductCode: '{34A7E999-C65F-4297-A523-3B08C6C37352}'
```

```suggestion
  ProductCode: '{34A7E999-C65F-4297-A523-3B08C6C37352}'
  AppsAndFeaturesEntries:
    - DisplayVersion: 5.9.0.4129
```
Marketing version is `5.9.0`, but DisplayVersion written to registry (visible in Apps&Features) that WinGet uses to match upgrades is `5.9.0.4129` and should be mentioned in the manifest to avoid upgrade issues